package.json

```diff
@@ -20,22 +20,23 @@
         "precommit": "npm run eslint && npm run stylelint",
         "test": "jest"
     },
-    "dependencies": {
+    "devDependencies": {
         "clean-css-cli": "4.2.1",
         "eslint": "5.1.0",
         "eslint-plugin-jest": "22.1.2",
-        "node-sass": "4.9.4",
-        "stylelint": "9.3.0",
-        "uglify-js": "2.8.29"
-    },
-    "devDependencies": {
         "gulp": "3.9.1",
         "gulp-sass": "4.0.1",
         "gulp-stylelint": "7.0.0",
         "gulp-watch": "5.0.0",
         "husky": "0.14.3",
         "jest": "23.6.0",
+        "node-sass": "4.9.4",
+        "react": "16.8.1",
```

Comment:
I don't think there is ever a case where you use react on the web without react-dom, so you might as well add that now, if you're adding react.  Also, you can't really use react without Babel, so consider adding that now.

Actually, why is react a dev dependency? When we start actually using it, it will be a regular dependency, and react-dom will be required too. Maybe you've got it here for some other reason, I guess.

I've got a wip branch with this set of devDependencies. I don't know if that is my final set or if it will need tweaks:

```
    "devDependencies": {
        "@babel/core": "^7.2.2",
        "@babel/plugin-proposal-class-properties": "^7.3.0",
        "@babel/preset-env": "^7.3.1",
        "@babel/preset-flow": "^7.0.0",
        "@babel/preset-react": "^7.0.0",
        "@emotion/babel-preset-css-prop": "^10.0.7",
        "babel-loader": "^8.0.5",
        "babel-plugin-emotion": "^10.0.7",
        "flow-bin": "^0.92.1",
        "gulp": "3.9.1",
        "gulp-sass": "4.0.1",
        "gulp-stylelint": "7.0.0",
        "gulp-watch": "5.0.0",
        "husky": "0.14.3",
        "jest": "23.6.0",
        "prettier": "^1.16.4",
        "sass-true": "4.0.0",
        "svgo": "1.0.5",
        "webpack": "^4.29.3",
        "webpack-cli": "^3.2.3"
    }
```

Reply:
This looks like a good list to me, and includes some things I'm missing like ``webpack-cli``.

What are your ``dependencies``?

Reply:
Also, I'm on my first day with ``react``, so that's why it is in dev dependencies.

Reply:
Here are my dependencies, though this is from a branch that doesn't have running code yet: 

```
    "dependencies": {
        "@emotion/core": "^10.0.7",
        "@emotion/styled": "^10.0.7",
        "clean-css-cli": "4.2.1",
        "eslint": "5.1.0",
        "eslint-plugin-jest": "22.1.2",
        "node-sass": "4.9.4",
        "react": "^16.8.1",
        "react-dom": "^16.8.1",
        "stylelint": "9.3.0",
        "uglify-js": "2.8.29"
    },
```